code/WorkInProgress/MechanicMadness.dm

```diff
@@ -620,8 +612,6 @@
 
 				SEND_SIGNAL(src,COMSIG_MECHCOMP_TRANSMIT_DEFAULT_MSG, null)
 				flick("comp_money1", src)
-
-				unReady(0)//Make it ready now.
```

Comment:
Previously you could hold a stack of credits, and mash-click the payment_comp (while toggling hands) to rapid-fire send signals.
Don't think this was ever abused. I didn't even know it could be done. Now 1 sec delay.